docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,39 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.
+
+**Subprojects** and **Cohorts** are terms used interchangeably in LORIS. **Subprojects** are defined in the `subproject` table of the database.
+
+
+## Adding Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page.
+  
+   You then click on _New SubprojectID_, fill in the fields on the right, click _save_. Refresh the page to view your new subproject. If you accidentally create an extra entry, you will have to delete it from the database manually with an sql command.  This process is similar to adding new projects.
+   
+> Note: Clicking _save_ more than once will register a duplicate subproject ID.
+
+### SQL
+Subprojects can be added directly in SQL using the following command. The `WindowDifference` and `RecruitmentTarget` are optional but should be set when the information is available.
+
+```sql
+INSERT INTO subproject
+(title, useEDC, WindowDifference, RecruitmentTarget)
+VALUES('SCI', 0, NULL, NULL);
+```
+
+
+### API
+ _not yet available. See [API documentation](../../../API/) for latest additions_
+ 
+
+## Interaction With LORIS
+
+### Projects
+**Subprojects** must be associated to Projects (when enabled, see [Projects setup](./Projects.md) for more details) in order to be able to create timepoints for a candidate. This association should be defined directly in SQL as follows
```

Comment:
the link on this line will probably not work. thats okai for now . untill we decide on how to organize the files to be most accessible
content is the important part for now

Reply:
@ridz1208 I believe you can have relative links in markup files with GitHub. You just need to remove the "./" and have it be (Projects.md) for it to work.

Reply:
indeed. I believe that is how we did this in LORIS-MRI in the docs directory.

Reply:
fixed